src/Resources/config/routing.yml

```diff
@@ -0,0 +1,14 @@
+list_tubes:
+  path:     /api/queue/tubes
+  defaults: { _controller: AppBundle:Queue:listTubes }
+  methods:  [GET]
+
+get_tube:
+  path:     /api/queue/{tube}
+  defaults: { _controller: AppBundle:Queue:getInfo }
+  methods:  [GET]
+
+list_all_info:
+  path:     /api/queue
+  defaults: { _controller: AppBundle:Queue:listInfo }
+  methods:  [GET]
```

Comment:
I'd recommend using XML to define your routes. Also: all of these names need to be prefixed with the `pmg_pheanstalk` namespace: `pmg_pheanstalk.status_tubes`, `pmg_pheanstalk.stats_tube`, etc.